src/libcollections/vec.rs

```diff
@@ -1296,6 +1296,17 @@ impl<T: PartialEq> Vec<T> {
     pub fn dedup(&mut self) {
         self.dedup_by(|a, b| a == b)
     }
+    
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Hi! This would be introduced as an unstable feature if introduced. This should be an `unstable` tag, and you specify a new feature name that you invent for `feature=`. See `dedup_by_key` in the same file for an example.